<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DanhSachMonHoc</class>
 <widget class="QDialog" name="DanhSachMonHoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>771</width>
     <height>531</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>651</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lblMaMH">
           <property name="text">
            <string>Mã môn học</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtMaMH">
           <property name="layoutDirection">
            <enum>Qt::LayoutDirection::LeftToRight</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblTenMH">
           <property name="text">
            <string>Tên môn học</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edtTenMH">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lblSTCLT">
           <property name="text">
            <string>Số tín chỉ LT</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spnSTCLT">
           <property name="maximum">
            <number>12</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSTCTH">
           <property name="text">
            <string>Số tín chỉ TH</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spnSTCTH">
           <property name="maximum">
            <number>12</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btnThemMH">
           <property name="text">
            <string>Thêm</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSuaMH">
           <property name="text">
            <string>Sửa</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnXoaMH">
           <property name="text">
            <string>Xóa</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tblMonHoc">
       <property name="columnCount">
        <number>8</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>162</number>
       </attribute>
       <column>
        <property name="text">
         <string>Mã</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Tên</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>SLTCLT</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>SLTCTH</string>
        </property>
       </column>
       <column/>
       <column/>
       <column/>
       <column/>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnQuayLai">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Quay lại</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
